DataFormats/Phase2TrackerDigi/interface/QCore.h

```diff
@@ -0,0 +1,60 @@
+#ifndef QCORE_H
```

Comment:
inclusion guard non conform to rule 4.1 http://cms-sw.github.io/cms_coding_rules.html#4--technical-coding-rules-1